litex_boards/targets/sipeed_tang_nano_20k.py

```diff
@@ -71,7 +72,7 @@ def __init__(self, sys_clk_freq=48e6,
         # TODO: XTX SPI Flash
 
         # SDR SDRAM --------------------------------------------------------------------------------
-        if not self.integrated_main_ram_size:
+        if with_sdram and not self.integrated_main_ram_size:
```

Comment:
ok will undo that. I think I may have tried `--integrated_main_ram_size`  and it yelled at me so I just added a flag.

Reply:
Tang Primer 25K has such an option already but I guess because you need to specify the model?

Reply:
Ah ok! I see why you have added this option. Tang 25k has no SDRAM, this option is to enable a SDRAM (sipeed or mister) connected to the header -> by default no SDRAM, if the user has plugged a module it may select the correct one